roles/defaults/vars/main.yml

```diff
@@ -130,7 +130,7 @@ _atmosphere_images:
   magnum_cluster_api_proxy: "registry.atmosphere.dev/library/magnum:{{ atmosphere_release }}"
   magnum_conductor: "registry.atmosphere.dev/library/magnum:{{ atmosphere_release }}"
   magnum_db_sync: "registry.atmosphere.dev/library/magnum:{{ atmosphere_release }}"
-  magnum_registry: quay.io/vexxhost/magnum-cluster-api-registry:latest
+  magnum_registry: quay.io/vexxhost/magnum-cluster-api-registry:main
```

Comment:
I think the `latest` is properly tagged https://harbor.atmosphere.dev/harbor/projects/7/repositories/vexxhost%2Fmagnum-cluster-api-registry/artifacts-tab/artifacts/sha256:d21f2b7a74555bd0ae7f6cde79106443e7dc07b7b2f44d578e82b38e09ab883f?sbomDigest=